packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/PageNames/SettingsName.tsx

```diff
@@ -51,20 +48,8 @@ export const SettingsName = () => {
         }
     };
 
-    const handleBackClick = () => dispatch(goto('suite-start'));
-
     return (
         <Container>
-            {isLoggedOut && (
-                <IconButton
-                    icon="caretLeft"
-                    variant="tertiary"
-                    size="medium"
-                    onClick={handleBackClick}
-                    data-testid="@settings/menu/close"
```

Comment:
this causes troubles in the E2E tests. The logical replacement is the `suite-start` menu item but it is not entirely 1:1. Coz the purpose of this button was to literally "close the settings".

Reply:
seems to be okay tho